include/app.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <aos/common/error.hpp>
+
+/**
+ * Aos zephyr application.
+ */
+class App {
+public:
+    /**
+     * Initializes application.
+     *
+     * @return aos::Error
+     */
+    aos::Error Init();
```

Comment:
Should we call this function in a constructor? In this case, we won't forget to call this method.

Reply:
We should not. As we don't use the exceptions all necessary initialization should be performed in `Init` function to have possibility to handle errors. See `main` function in this commit.